File: OOP/Week04/Tasks - Week 04/01_info.py
class Information:

    def __init__(self, name, address, age, phone_number):
        self.name = name
        self.address = address
        self.age = age
        self.phone_number = phone_number

    def get_name(self):
        return self.name

    def get_address(self):
        return self.address

    def get_age(self):
        return self.age

    def get_phone_number(self):
        return self.phone_number

    def set_name(self, name):
        self.name = name

    def set_address(self, address):
        self.address = address

    def set_age(self, age):
        self.age = age

    def set_phone_number(self, phone_number):
        self.phone_number = phone_number


info1 = Information("Badr", "Lahore, Pakistan", 20, "+92 1234567890")
info2 = Information("Adeen", "Karachi, Pakistan", 21, "+92 234567890")
info3 = Information("Azan", "Islamabad, Pakistan", 20, "+ 92234567890")

File: OOP/Week04/Tasks - Week 04/01_info.java
class Information {

    private String name;
    private String address;
    private int age;
    private String phoneNumber;
    
    public Information(String name1, String address1, int age1, String phoneNumber1) {
        name = name1;
        address = address1;
        age = age1;
        phoneNumber = phoneNumber1;
    }

    public String getName() {
        return name;
    }

    public String getAddress() {
        return address;
    }

    public int getAge() {
        return age;
    }
    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setName(String name1) {
        name = name1;
    }

    public void setAddress(String address1) {
        address = address1;
    }

    public void setAge(int age1) {
        age = age1;
    }
    public void setPhoneNumber(String phoneNumber1) {
        phoneNumber = phoneNumber1;
    }

}
